File: C/Problem Set 1/Mario/mario(less).c
//Mario
#include<stdio.h>
#include<cs50.h>
int main()
{
    int h;
    do
    {
        //Prompt user for input
        printf("Height: ");
        scanf("%d", &h);
    }
    while (h < 1 || h > 8);
    for (int i = 0; i < h; i++)
    {
        for (int j = h  - 1; j > i; j--)
        {
            printf(" ");
        }
        for (int j = i; j >= 0; j--)
        {
            printf("#");
        }
        printf("\n");
    }
    return 0;
}

File: Python/Problem Set 6/Mario/mario(less).py
import cs50

h = cs50.get_int("Height: ")

if (h < 1 or h > 8):
    while (h < 1 or h > 8):
        h = cs50.get_int("Height: ")

for i in range(h):
    for j in range(h-1, i, -1):
        print(" ", end="")
    for j in range(i, -1, -1):
        print("#", end="")
    print("")
